Changelog.md

```diff
@@ -2,8 +2,54 @@
 
 ## Gazebo Tools 2.x
 
+### Gazebo Tools 2.0.2 (2025-01-30)
+
+1. Add colcon.pkg with gz-cmake4 dependency
+    * [Pull request #142](https://github.com/gazebosim/gz-tools/pull/142)
+
+1. Fix dll search paths for Ruby on Windows
+    * [Pull request #140](https://github.com/gazebosim/gz-tools/pull/140)
+
+1. Add package.xml
+    * [Pull request #136](https://github.com/gazebosim/gz-tools/pull/136)
+
```

Comment:
I think this block above should be added below `### Gazebo Tools 2.X.X (20XX-XX-XX)` but looks like it's already like this in `gz-tools2`. We can fix it there